File: snippets/csharp/FiatCurrencies.cs
using Breez.Sdk.Liquid;

public class FiatCurrenciesSnippets
{
    public void ListFiatCurrencies(BindingLiquidSdk sdk)
    {
        // ANCHOR: list-fiat-currencies
        try
        {
            var fiatCurrencies = sdk.ListFiatCurrencies();
        }
        catch (Exception)
        {
            // Handle error
        }
        // ANCHOR_END: list-fiat-currencies
    }

    public void FetchFiatRates(BindingLiquidSdk sdk)
    {
        // ANCHOR: fetch-fiat-rates
        try
        {
            var fiatRates = sdk.FetchFiatRates();
        }
        catch (Exception)
        {
            // Handle error
        }
        // ANCHOR_END: fetch-fiat-rates
    }
}

File: snippets/python/src/fiat_currencies.py
from breez_sdk_liquid import BindingLiquidSdk

def list_fiat_currencies(sdk: BindingLiquidSdk):
   # ANCHOR: list-fiat-currencies
   try:
      supported_fiat_currencies = sdk.list_fiat_currencies()
   except Exception as error:
      print(error)
      raise
   # ANCHOR_END: list-fiat-currencies

def fetch_fiat_rates(sdk: BindingLiquidSdk):
   # ANCHOR: fetch-fiat-rates
   try:
      fiat_rates = sdk.fetch_fiat_rates()
   except Exception as error:
      print(error)
      raise
   # ANCHOR_END: fetch-fiat-rates
